staging/thanos-traefik/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for traefik-prometheus-thanos.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# Setting this option to true will allow installing this chart and not creating
+# any configuration objects.
+disable: false
```

Comment:
I find myself a little confused about what `disable: false` is actually doing, it feels like there are too many double negatives. perhaps `enabled: true` is more intuitive? (and I think follows the pattern in other values files)

Reply:
changed to `enabled: true`.